src/components/Searchbar.stories.js

```diff
@@ -0,0 +1,11 @@
+import Searchbar from './Searchbar';
+
+export default {
+  title: 'Components/Searchbar',
+  component: Searchbar,
+};
+
+const Template = args => <Searchbar {...args} />;
+
+export const Default = Template.bind({});
+Default.args = {};
```

Comment:
If you do not have any arguments, you can skip the template if you like:

```suggestion

export const Default = <Searchbar />
```